content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -280,20 +371,20 @@ mechanism.
 As mechanisms mature, they will be defined below on a per-policy basis. The methods of enforcement
 of individual policies are not defined here.
 
-[**PodSecurityPolicy**](/docs/concepts/policy/pod-security-policy/)
+[**PodSecurityPolicy**](/docs/concepts/profile/pod-security-profile/)
 
-- {{< example file="policy/privileged-psp.yaml" >}}Privileged{{< /example >}}
-- {{< example file="policy/baseline-psp.yaml" >}}Baseline{{< /example >}}
-- {{< example file="policy/restricted-psp.yaml" >}}Restricted{{< /example >}}
```

Comment:
suggest not changing these URLs?

Reply:
Looks like this was a remnant from a "Replace All" attempt..good catch :)